doc/release-notes/10810-search-api-payload-extensions.md

```diff
@@ -0,0 +1,52 @@
+Search API (/api/search) response will now include new fields for the different entities.
+
+For Dataverse:
+
+- "affiliation" 
+- "parentDataverseName"
+- "parentDataverseIdentifier"
+- "image_url" (optional)
+
+```javascript
+"items": [
+    {
+        "name": "Darwin's Finches",
+        ...
+        "affiliation": "Dataverse.org",
+        "parentDataverseName": "Root",
+        "parentDataverseIdentifier": "root",
+        "image_url":"data:image/png;base64,iVBORw0..."
+(etc, etc)
+```
+
+For DataFile:
+
+- "releaseOrCreateDate"
+- "image_url" (optional)
+
+```javascript
+"items": [
+    {
+        "name": "test.txt",
+        ...
+        "releaseOrCreateDate": "2016-05-10T12:53:39Z",
+        "image_url":"data:image/png;base64,iVBORw0..."
+(etc, etc)
+```
+
+For Dataset:
+
+- "image_url" (optional)
+
+```javascript
+"items": [
+    {
+        ...
+        "image_url": "http://localhost:8080/api/datasets/2/logo"
+        ...
+(etc, etc)
+```
+
+The image_url field was already part of the SolrSearchResult JSON, but it wasn’t returned by the API because it was appended only after the Solr query was executed in the SearchIncludeFragment of JSF. Now, the field is set in SearchServiceBean, ensuring it is always returned by the API when an image is available.
```

Comment:
```suggestion
The image_url field was already part of the SolrSearchResult JSON (and incorrectly appeared in Search API documentation), but it wasn’t returned by the API because it was appended only after the Solr query was executed in the SearchIncludeFragment of JSF. Now, the field is set in SearchServiceBean, ensuring it is always returned by the API when an image is available.
```